### PR TITLE
Navbar improvements and fixes

### PR DESCRIPTION
**Pull request by rishichawda**
- [x] Simplify navbar usage.
- [x] Fix responsiveness.
- [x] Fix navbar mobile view.

### CODE REVIEW

**Comment by rishichawda** [comment]
Earlier usage of Navbar was ( kind of ) complicated to be used in a responsive way. This PR is mainly focused on that.

Earlier usage : 
```
 const NavbarMenu = () => (
   <div className="navlinks">
     <span>home</span>
     <span>dashboard</span>
     <span>contact</span>
   </div>
 );

 const DesktopView = () => (
   <div className="navbarMenu">
     { <NavbarMenu /> }
   </div>
 );

 const MobileView = () => (
   <div className="navbarMobileMenu">
     <Popover content={<NavbarMenu />} position="bottomRight" noAction>
       <span className="icon-menu" />
     </Popover>
   </div>
 );

 ReactDOM.render(
   <Navbar leftIcon={<MenuIcon />} title="logo">
     <DesktopView />
     <MobileView />
   </Navbar>,
   document.getElementById('index'),
 );
```

Usage after merging : 

```
ReactDOM.render(
  <Navbar title="logo">
    <span>home</span>
    <span>dashboard</span>
    <span>contact</span>
  </Navbar>,
  document.getElementById('index'),
);
```

**Comment by prasanna1211** [comment]
@rishichawda How does the rendering happens on mobile view ? Will it always compress ?

**Comment by rishichawda** [comment]
> @rishichawda How does the rendering happens on mobile view ? Will it always compress ?

Yes, it switches to a dropdown view when on a smaller screen size. 

screenshot from 2018-12-03 16-33-43

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// import React from 'react';
-// import ReactDOM from 'react-dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (KRRISH96) [inline src/index.js]: Comment out these 2 lines.

File: src/index.js [hunk 2]
```diff
@@ -56,36 +56,12 @@ export default {
...
-//   document.getElementById('index'),
-// );
+ReactDOM.render(
+  <Navbar title="logo">
+    <span>home</span>
+    <span>dashboard</span>
+    <span>contact</span>
+  </Navbar>,
+  document.getElementById('index'),
+);
```
Comment (KRRISH96) [inline src/index.js]: Comment out these also.

File: src/navbar/theme.scss [hunk 3]
```diff
@@ -4,10 +4,8 @@
   z-index: 2;
   box-sizing: border-box;
   width: 100%;
-  height: 81px;
+  height: fit-content;
```
Comment (KRRISH96) [inline src/navbar/theme.scss:8]: We can provide a fallback to some fixed height since fit-content not supported in EDGE and IE

**Review by KRRISH96 — APPROVED** [review]

**Pull request merged**